api/http/handler/stacks/handler.go

```diff
@@ -54,6 +54,8 @@ func NewHandler(bouncer *security.RequestBouncer) *Handler {
 		bouncer.AuthenticatedAccess(httperror.LoggerHandler(h.stackDelete))).Methods(http.MethodDelete)
 	h.Handle("/stacks/{id}",
 		bouncer.AuthenticatedAccess(httperror.LoggerHandler(h.stackUpdate))).Methods(http.MethodPut)
+	h.Handle("/stacks/{id}/git",
+		bouncer.AuthenticatedAccess(httperror.LoggerHandler(h.stackGitUpdate))).Methods(http.MethodPost)
```

Comment:
using the same handler as in "redeploy git stacks" PR, will break that feature. as you use this route just to update config and the former feature uses this route to update the files and redeploy the stack.

Reply:
hmmmm, I may need to talk with Dmitry tomorrow